.gitignore

```diff
@@ -1,3 +1,5 @@
+# Use .git/info/exclude to ignore files related to your IDE
```

Comment:
Usual recommendation is to add your IDE to your computer gitignore, not to this file on each repo
Maybe a link to https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer would give more informations